.scrutinizer.yml

```diff
@@ -1,14 +0,0 @@
-build:
```

Comment:
@damianopetrungaro Why are you removing this file? I can see it is handling code coverage reports?

Reply:
I migrated from `scrutinizier` to `travis` (it is faster and way easier to interact with)
We can add the tests coverage using `travis` too.

I am free after 6/7 PM  to do it 😄